Forte.EpiServer.AzureSearch/Events/PageSearchEventHandler.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections.Generic;
+using System.Linq;
+using EPiServer;
+using EPiServer.Core;
+using Forte.EpiServer.AzureSearch.Extensions;
+using Forte.EpiServer.AzureSearch.Model;
+
+namespace Forte.EpiServer.AzureSearch.Events
+{
+    public class PageSearchEventHandler<T> where T : ContentDocument
+    {
+        private readonly IContentRepository _contentRepository;
```

Comment:
IContentLoader

Reply:
True, it's a leftover from before splitting the searchEventHandler class to page and block ones.